pkg/sidecarexec/sandbox_cmd_runner_darwin.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package sidecarexec
+
+import (
+	goexec "os/exec"
+
+	"github.com/vmware-tanzu/carvel-kapp-controller/pkg/exec"
+)
+
+func (SandboxCmdRunner) Run(cmd *goexec.Cmd, opts exec.RunOpts) error {
+	panic("Not implemented on this OS")
```

Comment:
We should mention this (not supported on OSX) somewhere in the docs / helper text of the flag

Reply:
the only reason why i added darwin is so that code compiles successfully on a mac. no intention to run sandboxing on a mac any time soon.